commons/com.b2international.index/src/com/b2international/index/revision/RevisionCompare.java

```diff
@@ -138,14 +144,16 @@ public RevisionCompare build() {
 	private final int totalAdded;
 	private final int totalChanged;
 	private final int totalRemoved;
+	private final int total;
```

Comment:
What is total here? Total `added` + `changed` + `removed`?

Reply:
No, it is the total, possible more than the sum of the three if it is more than the limit.